web-dev/src/components/data/Movie.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
The watched-list.css is empty right now but I've been using it to debug formatting and layout issues so it hasn't been empty. I will delete it if it's still empty by the end of the design phase.